plugins/basic/nexus-repository-nuget/src/test/java/com/sonatype/nexus/repository/nuget/internal/NugetGalleryFacetEntryTest.java

```diff
@@ -50,19 +52,24 @@ public void packageEntrySmokeTest() throws Exception {
     final String packageId = "screwdriver";
     final String version = "0.1.1";
 
-    final StorageTx tx = mock(StorageTx.class);
-    doReturn(tx).when(underTest).openStorageTx();
-
     // Wire the mocks together: component has asset, asset has format attributes
-    doReturn(component).when(underTest).findComponent(tx, packageId, version);
-    doReturn(asset).when(underTest).findAsset(tx, component);
+    doReturn(component).when(underTest).findComponent(any(StorageTx.class), eq(packageId), eq(version));
+    doReturn(asset).when(underTest).findAsset(any(StorageTx.class), eq(component));
     doReturn(mock(NestedAttributesMap.class)).when(asset).formatAttributes();
 
     final HashMap<String, ?> data = Maps.newHashMap();
     doReturn(data).when(underTest).toData(any(NestedAttributesMap.class),
         anyMapOf(String.class, String.class));
 
-    underTest.entry("base", packageId, version);
+    final StorageTx tx = mock(StorageTx.class);
+
+    UnitOfWork.beginBatch(tx);
```

Comment:
Reading this, I thought, 'Oh, is entry() a batch?' but I think it's not, it's just that beginBatch() is the easiest way to provide a mock tx to the infrastructure - is that correct?